src/Castle.Facilities.AspNetCore.Tests/Framework/Builders/ServiceProviderBuilder.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright 2004-2018 Castle Project - http://www.castleproject.org/
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace Castle.Facilities.AspNetCore.Tests.Framework.Builders
+{
+	using System;
+
+	using Microsoft.Extensions.DependencyInjection;
+
+	public class ServiceProviderBuilder
+	{
+		public static IServiceProvider New(IServiceCollection services)
+		{
+			return services.BuildServiceProvider(validateScopes: false);
```

Comment:
I would like to figure out what types are causing this to fail in the runtime tests @ https://github.com/fir3pho3nixx/Windsor.AspNetCore.

Reply:
Quick check revealed transients,